src/analysis/specific_limits.lean

```diff
@@ -560,6 +560,30 @@ begin
   exact eventually_norm_pow_le x,
 end
 
+/-- Bound for the sum of a geometric series in a normed ring.  This formula does not assume that the
+normed ring satisfies the axiom `∥1∥ = 1`. -/
+lemma normed_ring.tsum_geometric_of_norm_lt_1
+  (x : R) (h : ∥x∥ < 1) : ∥(∑' (n:ℕ), x ^ n)∥ ≤ ∥(1:R)∥ - 1 + (1 - ∥x∥)⁻¹ :=
+begin
+  let s : finset ℕ := {0},
```

Comment:
Can't this proof be streamlined using things like `tsum_eq_zero_add` or `summable_nat_add_iff`?

Reply:
Done, thanks for these pointers!